helix-view/Cargo.toml

```diff
@@ -42,6 +42,8 @@ log = "~0.4"
 
 which = "4.2"
 
+ec4rs = "1.0.1"
```

Comment:
One thing I am concern is the bus factor of ec4rs, there is only @TheDaemoness as the only maintainer that have edit access behind it, in case anything happens in the future we need to have a fork, not a deal breaker I guess. Maybe put it within helix-editor but the coding style seemed different than usual rust projects?

Reply:
Can you please add rust-bus or maybe someone else as a separate maintainer so that there is a chance to revive the project in case anything happens? But yeah, there can always be a fork. But I guess not much of a concern here.